src/main/java/io/quarkus/search/app/yml/QuarkusMetadata.java

```diff
@@ -18,6 +18,13 @@
 
 public class QuarkusMetadata {
 
+    private static final Map<String, String> TYPE_MAPPER = Map.of(
```

Comment:
I'm sorry, what is this?

Reply:
Ah, that's whay your explained here, I guess:

> I've also mapped the guide types to what the quarkus.io is using as otherwise the icons next to the search results do not match what is displayed initially 🙈

Still, I don't understand why this is needed... I'd expect our data to be authoritative, and if there's some transformation going on to turn it into a class name, well... it should happen on the presentation side, e.g. in the vue.js app or LitElement webcomponent.

Reply:
https://github.com/quarkusio/quarkusio.github.io/blob/bdff06f6bd1848dcf1b616ed14529f999e19246a/_includes/index-docs.html#L114-L123

^ see how a `type=guide` is a `class="referencebkg"` ... so this is to address this mismatch. Alternatively, we change it on the `quarkusio` side and use the classes that would actually map to the types

Reply:
> ^ see how a `type=guide` is a `class="referencebkg"` ... so this is to address this mismatch. Alternatively, we change it on the `quarkusio` side and use the classes that would actually map to the types

Yes, I would much rather have that. We shouldn't incorporate logic related to CSS classes in a rest service.